x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/apikey/ApiKeyRestIT.java

```diff
@@ -577,7 +578,7 @@ public void testGetPrivilegesForApiKeyThrows400IfItHasAssignedPrivileges() throw
         );
     }
 
-    public void testRemoteIndicesNotSupportedForApiKeys() throws IOException {
+    public void testRemoteIndicesSupportForApiKeys() throws IOException {
```

Comment:
This is a note for later (doesn't matter for the current draft state of the PR) but we will want to specifically test a scenario in which we create an API key without assigned privileges (and similarly when we update an API key to not have assigned roles anymore). In this case, the role descriptor intersection we get should only contain one set (not two). I'm 99% sure this works out of the box, but we'll want to ensure this is the case since the fulfilling cluster will incorrectly restrict permissions. Just jotting this down so I don't forget -- we can discuss the details during the sync.